packages/trader/src/Modules/Page404/Components/__tests__/Page404.spec.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import { render } from '@testing-library/react';
+import Page404 from '../Page404';
+
+describe('Page404', () => {
+    const browser_history = createBrowserHistory();
+
+    it('should render Page404', () => {
+        const { container } = render(
+            <Router history={browser_history}>
```

Comment:
@kate-deriv <Router container with browser_history is not necessary here. we can render Page404 as is🙂